src/applications/caregivers/tests/e2e/1010cg.cypress.spec.js

```diff
@@ -0,0 +1,164 @@
+import path from 'path';
+
+import testForm from 'platform/testing/e2e/cypress/support/form-tester';
+import { createTestConfig } from 'platform/testing/e2e/cypress/support/form-tester/utilities';
+
+import formConfig from '../../config/form';
+import manifest from '../../manifest.json';
+
+const veteranLabel = `Enter Veteran's or service member\u2019s full name`;
```

Comment:
`'Enter Veteran\u2019s ...'`?

Reply:
Yeah this is not ideal, using this as the labels for each field, have another ticket to refactor this as well as the switch statement inside the checkbox components.